poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
+              </div>
+            </div>
+            <div class="poll-percent-container">
+                <span class="poll-percent-display{{#if first}} poll-top-choice{{/if}}">{{percent}}%</span>
+            </div>
+        </li>
+       {{^last}}
+        <li class="poll-spacer">
```

Comment:
It would be nice if that would be possible in all cases.  This is using `display: table`, and it turns out that CSS doesn't allow to add space between table rows – margin and border are simply ignored at least by Webkit browsers.  I tried recreating the same layout with `display: inline-block`, but there are many things I can't even get to work on a single browser the same way it was before, let alone on all supported browsers.

How bad is this for screen readers?  It will likely be at least half a day's work to get this fixed properly.